.github/workflows/pull-request.yml

```diff
@@ -2,11 +2,6 @@ name: PR build
 
 on:
   pull_request:
```

Comment:
How about using `on: [pull_request]` since we don't have any branches to specify?

Reply:
Fixed in 5efad7a